### ISSUE TITLE
Specify number of items in array

### ISSUE THREAD

**Issue by chookie** [APIs-guru/graphql-faker#12]
Would it be possible to add the ability to specify the number of items in a list?  I would imagine being able to enter a min number and max number then it returns a random number of items within that range.  If the numbers are the same then obviously it just returns that number.  An example of syntax is below but you might have a better idea.

![image](https://cloud.githubusercontent.com/assets/4765327/26391449/c08af50e-405a-11e7-89a5-1719a22c63aa.png)

**Comment by IvanGoncharov** [comment]
@chookie Thank you for the idea it's definitely something we need to add to Faker.
It partly overlaps with validation tool we plan to work on so you could specify `@list(minItems: 5, maxIterms: 10)` for input value and we want to reuse the same directives inside Faker.

We still didn't finish with the design of these directives so it will take some time before we can add this feature to Faker.

**Comment by zachzhi** [comment]
Hello, @IvanGoncharov 
This list length will be very useful for us ;)
When we can have this feature?

**Comment by IvanGoncharov** [comment]
@zachzhi I plan to spend some time on Faker in the second half of this month and this issue is on my checklist. Hower before implementing it I want to switch to official `graphql-js` instead of the custom fork and write some basic tests.

I will ping you when I will have some prototype to test.

**Comment by OlzhasAlexandrov** [comment]
Any update on this?
Thank you for the tool, looks very promising 👍

**Comment by IvanGoncharov** [comment]
@OlzhasAlexandrov I'm still working on pushing necessary functionality in the graphql-js. It's extremely slow and time-consuming 😞 but I managed to push like 80% required for `APIs-guru/graphql-*` projects. 
For this particular project, I still need to push one PR that will allow switching away from the custom fork of `express-graphql`.

TL;DR; I'm working on my checklist but its very very slow progress.

**Comment by idkjs** [comment]
@OlzhasAlexandrov happy to help. I might learn something. Need/want help?

**Comment by lucasfeliciano** [comment]
I would be glad to help also, there is any todo list/PR/branch/fork that I can look into?

**Comment by IvanGoncharov** [comment]
@kuncevic Still planning to implement this, it's in my queue.

> May be you can pint out to the related line of code where I just can hard-code the number of items so I can build custom version of faker just to be able to play with long lists.

https://github.com/APIs-guru/graphql-faker/blob/2f58a7f4fc230609756b97f16bd3615df1c162d4/src/fake_schema.ts#L182

**Comment by kuncevic** [comment]
Just modified `node_modules\graphql-faker\dist\fake_schema.js` at `var length = fake_1.getRandomInt(500, 1000);` for now

**Comment by MilosMosovsky** [comment]
Hello everyone, I just created #61 as I desperately needs this feature and hardcoding is no-go in my case. Check it out :) 

My proposal is following

`employees: [Employee] @sample(min: 10, max: 20)` 

/cc: @IvanGoncharov

**Comment by kristianmandrup** [comment]
Why call it `@sample`? why not sth like `@itemCount` or `@items`? Otherwise looks neat :)

**Comment by kuncevic** [comment]
> Why call it `@sample`? why not sth like `@itemCount` or `@items`? Otherwise looks neat :)

I had same thought, but I would stick to @fakeArray

**Comment by sneko** [comment]
It's been a while since last posts 😄 ... but wondering if you are aware of something that fills the need?

Thank you,
